try.html

```diff
@@ -117,16 +117,16 @@ <h3 id="build"> Build </h3>
     <td><code>https://img.shields.io/magnumci/ci/96ffb83fa700f069024921b0702e76ff/new-meta.svg</code></td>
   </tr>
   <tr><th> CircleCI: </th>
-      <td><img src='/circleci/project/BrightFlair/PHP.Gt.svg' alt=''/></td>
-      <td><code>https://img.shields.io/circleci/project/BrightFlair/PHP.Gt.svg</code></td>
+      <td><img src='/circleci/project/github/RedSparr0w/node-csgo-parser.svg?maxAge=2592000' alt=''/></td>
```

Comment:
Could you remove maxAge from try.html please?

index.html is meant to be generated from try.html through `make website`, which I can do for you.